src/migration/1726556744920-SourceCategorySlug.ts

```diff
@@ -0,0 +1,40 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class SourceCategorySlug1726556744920 implements MigrationInterface {
+  name = 'SourceCategorySlug1726556744920';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "source_category" ADD "slug" text GENERATED ALWAYS AS (trim(BOTH '-' FROM regexp_replace(lower(trim(COALESCE(LEFT(source_category.title,100),'')||'-'||source_category.id)), '[^a-z0-9-]+', '-', 'gi'))) STORED NOT NULL`,
```

Comment:
We need to expose slug in graphql type as well?

Reply:
Yes. Should be part of the Schema now.